README.md

```diff
@@ -1,28 +1,32 @@
 # FizzySteamyMirror
 
-This is a community maintained repo forked from [RayStorm](https://github.com/Chykary/FizzySteamyMirror). Mirror [docs](https://mirror-networking.com/docs/Transports/Fizzy.html).
+This is a community maintained repo forked from **[RayStorm](https://github.com/Chykary/FizzySteamyMirror)**. 
 
-FizzySteamyMirror brings together [Steam](https://store.steampowered.com/) and [Mirror](https://github.com/vis2k/Mirror) utilising Async of a Steam P2P network transport layer for **Mirror**.
+Mirror **[docs](https://mirror-networking.com/docs/Transports/Fizzy.html)** and the official community **[Discord](https://discord.gg/N9QVxbM)**.
+
+FizzySteamyMirror brings together **[Steam](https://store.steampowered.com)** and **[Mirror](https://github.com/vis2k/Mirror)** utilising Async of a Steam P2P network transport layer for **Mirror**.
 
 ## Dependencies
-**Note: SteamWorks.NET is packed with this transport.**

```

Comment:
This would be confusing for someone who is just starting. If they was to download the git straight from GitHub they would have to install **SteamWorks.net** by themselves.